src/src/DataStructs/ControllerSettingsStruct.cpp

```diff
@@ -20,14 +20,14 @@ ControllerSettingsStruct::ControllerSettingsStruct()
 }
 
 void ControllerSettingsStruct::reset() {
-  UseDNS                     = false;
-  Port                       = 0;
+  UseDNS                     = DEFAULT_SERVER_USEDNS;
+  Port                       = DEFAULT_PORT;
```

Comment:
for MQTT the port is 1883 so we could use that as the default value.
but this will only be a rename as this constant is only used for the Controller settings. I merely added during the reset process to be consistent

Reply:
If a rename reduces confusion, then that's fine 😃

Reply:
ok

Reply:
Done